apollo-api/src/main/java/com/spotify/apollo/RequestValue.java

```diff
@@ -55,23 +56,27 @@ private static Request create(
       Map<String, List<String>> parameters,
       Map<String, String> headers,
       Optional<String> service,
-      Optional<ByteString> payload) {
+      Optional<ByteString> payload,
+      Optional<Duration> ttl) {
     return new AutoValue_RequestValue(
         method, uri,
         ImmutableMap.copyOf(parameters),
         ImmutableMap.copyOf(headers),
         service,
-        payload);
+        payload,
+        ttl);
   }
 
+  public abstract Optional<Duration> ttl();
```

Comment:
Should have an ```@Override``` annotation to ensure that it doesn't accidentally become an additional parameter.